SU-SWS/decanter
# Type Styles

# READY FOR REVIEW

# Summary
- Add missing type styles
- Removed unnecessary typography mixins and variables
- Misc. cleanup

# Needed By (Date)
- 5.0 release

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Compile styleguide
3. Ensure that the typography styles under Elements match those in Figma; ensure code structure and usage is OK

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
